crates/sovereign-sdk/module-system/sov-modules-stf-blueprint/src/lib.rs

```diff
@@ -497,51 +477,7 @@ where
         let checkpoint =
             self.begin_slot(checkpoint, slot_header, validity_condition, pre_state_root);
 
-        // Initialize batch workspace
-        let mut batch_workspace = checkpoint.to_revertable();
-        let mut kernel_working_set =
-            KernelWorkingSet::from_kernel(&self.kernel, &mut batch_workspace);
-        let selected_blobs = self
-            .kernel
-            .get_blobs_for_this_slot(blobs, &mut kernel_working_set)
-            .expect("blob selection must succeed, probably serialization failed");
-
-        info!(
-            "Selected {} blob(s) for execution in current slot",
-            selected_blobs.len()
-        );
-
-        let mut checkpoint = batch_workspace.checkpoint();
-
-        let mut batch_receipts = vec![];
-
-        for (blob_idx, mut blob) in selected_blobs.into_iter().enumerate() {
-            let (apply_blob_result, checkpoint_after_blob) =
-                self.apply_blob(checkpoint, blob.as_mut_ref());
```

Comment:
can use blobs here in place for selectted blobs, i assume this breaks sov-sdk tests